src/ui/ListItem.tsx

```diff
@@ -0,0 +1,45 @@
+/** @license
+ *  Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+
+import * as React from "react";
+
+import { CSS } from "../lib/Constants";
+
+/**
+ * Interface for the properties assigned to the ListItem component.
+ * @interface
+ */
+interface IListItemProps {
```

Comment:
Similarly, I think Google Style (and TypeScript convention) is to omit the `I` prefix.  If you've done it other places, you can do it here, but it may be worth doing a pass where you move this project's conventions closer to Google Style.